app/scripts/modules/cluster/reports/forms/cluster.project.form.details.js

```diff
@@ -759,6 +781,43 @@ angular.module( 'ngm.widget.project.details', [ 'ngm.provider' ])
 						}
 					});
 
+					//SAVE COMPONENT-PLANS
+					
+					 $scope.project.definition.plan_component = [];
+
+					 	if ($scope.project.definition.hrp_plan == true) {
+						  $scope.project.definition.plan_component.push('hrp_plan');
+						}
+
+						if ($scope.project.definition.rmrp_plan == true) {
+						  $scope.project.definition.plan_component.push('rmrp_plan');
+						}
+
+						if ($scope.project.definition.interagencial_plan == true) {
+						  $scope.project.definition.plan_component.push('interagencial_plan');
+						}
+
+						if ($scope.project.definition.humanitarian_component == true) {
+						  $scope.project.definition.plan_component.push('humanitarian_component');
+						}
+
+						if ($scope.project.definition.construccion_de_paz_component == true) {
+						  $scope.project.definition.plan_component.push('construccion_de_paz_component');
+						}
+
+						if ($scope.project.definition.desarrollo_sostenible_component == true) {
+						  $scope.project.definition.plan_component.push('desarrollo_sostenible_component');
+						}
```

Comment:
what if put it into a separate function, specific for Colombia, so that to keep _save_ function cleaner?